utils/codegen/asn1ToConversionHeader-rgen.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+
+# ==============================================================================
+# MIT License
+#
+# Copyright (c) 2023 Institute for Automotive Engineering (ika), RWTH Aachen University
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+# ==============================================================================
+
+import argparse
+import glob
+import os
+import re
+import shutil
+import subprocess
+import tempfile
+
+def parseCli():
+    """Parses script's CLI arguments.
+
+    Returns:
+        argparse.Namespace: arguments
+    """
+
+    parser = argparse.ArgumentParser(
+        description="Creates ROS .msg files from ASN1 definitions.",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    parser.add_argument("files", type=str, nargs="+", help="ASN1 files")
+    parser.add_argument("-o", "--output-dir", type=str, required=True, help="output directory")
+    parser.add_argument("-td", "--temp-dir", type=str, default=None, help="temporary directory for mounting files to container; uses tempfile by default")
+    parser.add_argument("-t", "--message", type=str, required=True, help="message name")
+    parser.add_argument("-di", "--docker-image", type=str, default="ghcr.io/ika-rwth-aachen/etsi_its_messages:rgen", help="rgen Docker image")
+
+    args = parser.parse_args()
+
+    return args
+
+def main():
+
+    args = parseCli()
+
+    # create output directory
+    os.makedirs(args.output_dir, exist_ok=True)
+
+    # create temporary directories for running asn1c in docker container
```

Comment:
There's some copy-paste comments here in this file that still need to be adjusted.

Reply:
Fixed.